README.md

```diff
@@ -1,15 +1,39 @@
 
-# Gatsby French Translation
+# Traduction française de Gatsby
 
-This repo contains the French translation for Gatsby.
+Ce répertoire contient la traduction française de Gatsby.
 
-Please refer to the [Translation Progress Issue](https://github.com/gatsbyjs/gatsby-fr/issues/1) to start translating!
+Merci d'aller jeter un oeil ici  [Translation Progress Issue](https://github.com/gatsbyjs/gatsby-fr/issues/1) pour choisir votre page et vous assurer que personne n'a déjà entamé sa traduction. Ne prenez qu'une page à la fois, même si vous êtes gourmands ! 😄 
 
-Useful Links:
+Quelques liens utiles :
 
 * [Style Guide](/style-guide.md)
-* [Gatsby Translation Guide](https://www.gatsbyjs.org/contributing/gatsby-docs-translation-guide/)
-* [Gatsby Discord](https://gatsby.dev/discord)
+* [Gatsby Translation Guide](https://www.gatsbyjs.org/contributing/gatsby-docs-translation-guide/) : idéalement à lire avant de vous lancer dans l'aventure. 
+* [Liste des termes communs](/common-terms.md) : liste évolutive des termes qui peuvent poser un problème de traduction. 
+* [Gatsby Discord](https://gatsby.dev/discord) : rejoignez-nous pour discuter librement dans le salon dédié : `localization-fr`. 
 
-(Feel free to translate this document and add any content you feel would be useful to contributors).
-  
\ No newline at end of file
+## Comment traduire une page ? 
+
+Plusieurs possibilités s'offrent à vous. 
+
+__Si vous n'avez jamais contribué à Gatsby__, faites un fork de ce répertoire. Vous pourrez ensuite éditer le fichier directement dans l'éditeur proposé par GitHub. 
+
+Une fois terminé, faites un _pull-request_. Votre traduction sera alors examinée par un ou plusieurs mainteneurs du projet qui pourront à ce moment-là proposer des modifications à apporter à votre traduction concernant l'orthographe, la syntaxe ou la terminologie qui peut parfois être complexe à traduire. 
```

Comment:
I think pull request is feminine, isn’t it ?

Reply:
yep you're right